tests/FakeItEasy.Specs/ArgumentMatchingSpecs.cs

```diff
@@ -326,7 +326,7 @@ public static void StronglyTypedWhenArgumentsMatchWith4ArgsWrongSignature(IFoo f
 
             "And a fake method with 4 parameters is configured with an arguments predicate with an incompatible signature"
                 .x(() => exception = Record.Exception(
-                        () => A.CallTo(() => fake.Bar(0, null, false, null))
+                        () => A.CallTo(() => fake.Bar(0, string.Empty, false, new object()))
                             .WhenArgumentsMatch((long a, DateTime b, Type c, char d) => true)
                             .Returns(42)));
 

```

Comment:
I'm not sure there's much value in those changes. Why not mark the parameters as nullable in the interface? In this situation we don't actually care if the values are null.

Reply:
Mostly because I didn't want to pull in the language change yet. If you think the time is right, I can take that tack instead.